superset-frontend/packages/superset-ui-chart-controls/src/shared-controls/components/ColumnConfigControl/constants.tsx

```diff
@@ -142,6 +143,15 @@ const colorPositiveNegative: ControlFormItemSpec<'Checkbox'> = {
   debounceDelay: 200,
 };
 
+const truncateLongCells: ControlFormItemSpec<'Checkbox'> = {
+  controlType: 'Checkbox',
+  label: t('Truncate Cells'),
+  description: t('Truncate long cells to the "min width" set above'),
+  width: 150,
```

Comment:
Does change the width here fix the jumpy info tooltip icon? (It should be on the same line as the checkbox label)

Reply:
I noticed that too, I'll check

Reply:
I removed the width line, it's unnecessary. I copied that over from something else I had tweaked in that control that did need a narrower width.